examples/visualization/draw_segmentations_demo.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A demo for visualization of segmentation masks on flower
+dataset i.e `oxford_flowers102`.
+
+Demonstrate visualization of centered rectangle masks on batch
+of flower images using `draw_segmentation` utility of keras_cv.
+Plot the figure using `matplotlib`
+"""
+
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+from keras_cv.visualization import draw_segmentation
```

Comment:
Let's not import individual names, just use keras_cv.visualization.draw_segmentation